runtime/parachains/src/configuration.rs

```diff
@@ -192,8 +192,6 @@ pub struct HostConfiguration<BlockNumber> {
 	pub dispute_period: SessionIndex,
 	/// How long after dispute conclusion to accept statements.
 	pub dispute_post_conclusion_acceptance_period: BlockNumber,
-	/// The maximum number of dispute spam slots
-	pub dispute_max_spam_slots: u32,
```

Comment:
Won't this need a migration too?

Reply:
I recall that this struct is stored somewhere onchain.

Reply:
There is a migration for this already in this PR (configuration).

Reply:
Yes, it is in `configuration/migration.rs` (the only reason I touched this file).